### ISSUE TITLE
deploy fails when using sass

### ISSUE THREAD

**Issue by jmonster** [tonycoco/heroku-buildpack-ember-cli#36]
I'm trying to push out an app that is barely more than static html -- I've installed `broccoli-sass` into my project.

There seems to be a missing dependency in the build-pack to let this compile properly?

```
-----> Installing dependencies

       > node-sass@1.1.4 install /tmp/build_1e55a9301b11e6cf6155c90e55124480/node_modules/broccoli-sass/node_modules/node-sass
       > node build.js

       `linux-x64-v8-3.14` exists; testing
       make: Entering directory `/tmp/build_1e55a9301b11e6cf6155c90e55124480/node_modules/broccoli-sass/node_modules/node-sass/build'
         CXX(target) Release/obj.target/binding/binding.o
       cc1plus: error: unrecognized command line option "-std=c++11"
       cc1plus: error: unrecognized command line option "-std=c++11"
       make: *** [Release/obj.target/binding/binding.o] Error 1
       make: Leaving directory `/tmp/build_1e55a9301b11e6cf6155c90e55124480/node_modules/broccoli-sass/node_modules/node-sass/build'
       gyp ERR! build error
       gyp ERR! stack Error: `make` failed with exit code: 2
       gyp ERR! stack     at ChildProcess.onExit (/tmp/build_1e55a9301b11e6cf6155c90e55124480/vendor/node/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
       gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
       gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:810:12)
       gyp ERR! System Linux 3.8.11-ec2
       gyp ERR! command "node" "/tmp/build_1e55a9301b11e6cf6155c90e55124480/vendor/node/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
       gyp ERR! cwd /tmp/build_1e55a9301b11e6cf6155c90e55124480/node_modules/broccoli-sass/node_modules/node-sass
       gyp ERR! node -v v0.10.33
       gyp ERR! node-gyp -v v1.0.1
       gyp ERR! not ok
       Build failed
       npm ERR! node-sass@1.1.4 install: `node build.js`
       npm ERR! Exit status 1
       npm ERR!
       npm ERR! Failed at the node-sass@1.1.4 install script.
       npm ERR! This is most likely a problem with the node-sass package,
       npm ERR! not with npm itself.
       npm ERR! Tell the author that this fails on your system:
       npm ERR!     node build.js
       npm ERR! You can get their info via:
       npm ERR!     npm owner ls node-sass
       npm ERR! There is likely additional logging output above.

       npm ERR! System Linux 3.8.11-ec2
       npm ERR! command "/tmp/build_1e55a9301b11e6cf6155c90e55124480/vendor/node/bin/node" "/tmp/build_1e55a9301b11e6cf6155c90e55124480/vendor/node/bin/npm" "install" "--quiet" "--userconfig" "/tmp/build_1e55a9301b11e6cf6155c90e55124480/.npmrc"
       npm ERR! cwd /tmp/build_1e55a9301b11e6cf6155c90e55124480
       npm ERR! node -v v0.10.33
       npm ERR! npm -v 1.4.28
       npm ERR! code ELIFECYCLE
       npm ERR! not ok code 0

 !     Push rejected, failed to compile Ember CLI app
```

**Comment by zzarcon** [comment]
Hi guys, i tried the proposal of @jmonster about using cedar-14 but I still get the error. Can anyone help me? thanks

**Comment by tonycoco** [comment]
Can you paste the stack trace?

**Comment by mikekellyio** [comment]
I'm running into heroku failing to build this on ``cedar-14`` as well. Below is my stack trace:

```
Counting objects: 158, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (136/136), done.
Writing objects: 100% (158/158), 2.59 MiB | 159.00 KiB/s, done.
Total 158 (delta 26), reused 0 (delta 0)
remote: Compressing source files... done.
remote: Building source:
remote:
remote: -----> Fetching custom git buildpack... done
remote: -----> Ember CLI app detected
remote: -----> Exporting config vars to environment
remote: -----> Building environment: production
remote:
remote:        PRO TIP: Avoid using semver ranges starting with '>' in engines.node
remote:        See https://devcenter.heroku.com/articles/nodejs-support
remote:
remote: -----> Requested node range:  >= 0.10.0
remote: -----> Resolved node version: 4.2.1
remote: -----> Downloading and installing node
remote: -----> Using default npm version: 2.14.7
remote: -----> Downloading and installing nginx
remote: -----> Adding boot script
remote: -----> Copying configs
remote: -----> Forcing rebuild of all node_modules. Pruning all node_modules.
remote: -----> Forcing re-installation of all bower_components. Pruning all bower_components.
remote: -----> Creating node_modules cache directory
remote: -----> Creating bower_components cache directory
remote: -----> Installing bower which is required by other dependencies
remote:        bower@1.5.3 node_modules/bower
remote:        ├── is-root@1.0.0
remote:        ├── junk@1.0.2
remote:        ├── stringify-object@1.0.1
remote:        ├── chmodr@0.1.0
remote:        ├── user-home@1.1.1
remote:        ├── q@1.4.1
remote:        ├── abbrev@1.0.7
remote:        ├── opn@1.0.2
remote:        ├── bower-endpoint-parser@0.2.2
remote:        ├── bower-logger@0.2.2
remote:        ├── lockfile@1.0.1
remote:        ├── archy@1.0.0
remote:        ├── nopt@3.0.4
remote:        ├── graceful-fs@3.0.8
remote:        ├── lru-cache@2.7.0
remote:        ├── retry@0.6.1
remote:        ├── tmp@0.0.24
remote:        ├── semver@2.3.2
remote:        ├── request-progress@0.3.1 (throttleit@0.0.2)
remote:        ├── p-throttler@0.1.1 (q@0.9.7)
remote:        ├── md5-hex@1.1.0 (md5-o-matic@0.1.1)
remote:        ├── shell-quote@1.4.3 (array-reduce@0.0.0, array-map@0.0.0, array-filter@0.0.1, jsonify@0.0.0)
remote:        ├── fstream@1.0.8 (inherits@2.0.1, graceful-fs@4.1.2)
remote:        ├── chalk@1.1.1 (escape-string-regexp@1.0.3, ansi-styles@2.1.0, supports-color@2.0.0, strip-ansi@3.0.0, has-ansi@2.0.0)
remote:        ├── bower-json@0.4.0 (intersect@0.0.3, deep-extend@0.2.11, graceful-fs@2.0.3)
remote:        ├── promptly@0.2.0 (read@1.0.7)
remote:        ├── which@1.2.0 (is-absolute@0.1.7)
remote:        ├── mkdirp@0.5.0 (minimist@0.0.8)
remote:        ├── fstream-ignore@1.0.2 (inherits@2.0.1, minimatch@2.0.10)
remote:        ├── glob@4.5.3 (inherits@2.0.1, once@1.3.2, inflight@1.0.4, minimatch@2.0.10)
remote:        ├── rimraf@2.4.3 (glob@5.0.15)
remote:        ├── github@0.2.4 (mime@1.3.4)
remote:        ├── tar-fs@1.8.1 (pump@1.0.0, tar-stream@1.2.2)
remote:        ├── request@2.53.0 (forever-agent@0.5.2, aws-sign2@0.5.0, caseless@0.9.0, tunnel-agent@0.4.1, oauth-sign@0.6.0, stringstream@0.0.4, isstream@0.1.2, json-stringify-safe@5.0.1, tough-cookie@2.2.0, qs@2.3.3, node-uuid@1.4.3, form-data@0.2.0, mime-types@2.0.14, combined-stream@0.0.7, http-signature@0.10.1, bl@0.9.4, hawk@2.3.1)
remote:        ├── cardinal@0.4.4 (ansicolors@0.2.1, redeyed@0.4.4)
remote:        ├── bower-registry-client@0.3.0 (async@0.2.10, graceful-fs@2.0.3, request-replay@0.2.0, lru-cache@2.3.1, rimraf@2.2.8, mkdirp@0.3.5, request@2.51.0)
remote:        ├── decompress-zip@0.1.0 (mkpath@0.1.0, touch@0.0.3, readable-stream@1.1.13, binary@0.3.0)
remote:        ├── update-notifier@0.3.2 (is-npm@1.0.0, string-length@1.0.1, semver-diff@2.0.0, latest-version@1.0.1)
remote:        ├── handlebars@2.0.0 (optimist@0.3.7)
remote:        ├── insight@0.7.0 (async@1.4.2, object-assign@4.0.1, tough-cookie@2.2.0, lodash.debounce@3.1.1, configstore@1.2.1, os-name@1.0.3, inquirer@0.10.1)
remote:        ├── mout@0.11.0
remote:        ├── bower-config@0.6.1 (osenv@0.0.3, graceful-fs@2.0.3, optimist@0.6.1, mout@0.9.1)
remote:        ├── configstore@0.3.2 (object-assign@2.1.1, xdg-basedir@1.0.1, uuid@2.0.1, osenv@0.1.3, js-yaml@3.4.3)
remote:        └── inquirer@0.8.0 (ansi-regex@1.1.1, figures@1.4.0, mute-stream@0.0.4, through@2.3.8, lodash@2.4.2, readline2@0.1.1, chalk@0.5.1, rx@2.5.3, cli-color@0.3.3)
remote: -----> Pruning cached dependencies not specified in package.json
remote: -----> Pruning cached bower dependencies not specified in bower.json
remote: -----> Installing dependencies
remote:        npm WARN deprecated pangyp@2.3.2: use node-gyp@3+, it does all the things
remote:
remote:        > node-sass@2.1.1 install /tmp/build_8d02b095bd87a9217ee50832ded9951d/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/node-sass
remote:        > node scripts/install.js
remote:
remote:        Can not download file from https://raw.githubusercontent.com/sass/node-sass-binaries/v2.1.1/linux-x64-node-4.2/binding.node
remote:
remote:        > node-sass@2.1.1 postinstall /tmp/build_8d02b095bd87a9217ee50832ded9951d/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/node-sass
remote:        > node scripts/build.js
remote:
remote:        make: Entering directory `/app/tmp/cache/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/node-sass/build'
remote:          CXX(target) Release/obj.target/binding/src/binding.o
remote:        In file included from ../src/binding.cpp:1:0:
remote:        ../node_modules/nan/nan.h:324:27: error: redefinition of ‘template<class T> v8::Local<T> Nan::imp::NanEnsureHandleOrPersistent(const v8::Local<T>&)’
remote:           NAN_INLINE v8::Local<T> NanEnsureHandleOrPersistent(const v8::Local<T> &val) {
remote:                                   ^
remote:        ../node_modules/nan/nan.h:319:17: error: ‘template<class T> v8::Handle<T> Nan::imp::NanEnsureHandleOrPersistent(v8::Handle<T>&)’ previously declared here
remote:           v8::Handle<T> NanEnsureHandleOrPersistent(const v8::Handle<T> &val) {
remote:                         ^
remote:        ../node_modules/nan/nan.h:344:27: error: redefinition of ‘template<class T> v8::Local<T> Nan::imp::NanEnsureLocal(v8::Handle<T>&)’
remote:           NAN_INLINE v8::Local<T> NanEnsureLocal(const v8::Handle<T> &val) {
remote:                                   ^
remote:        ../node_modules/nan/nan.h:334:27: error: ‘template<class T> v8::Local<T> Nan::imp::NanEnsureLocal(const v8::Local<T>&)’ previously declared here
remote:           NAN_INLINE v8::Local<T> NanEnsureLocal(const v8::Local<T> &val) {
remote:                                   ^
remote:        ../node_modules/nan/nan.h:757:13: error: ‘node::smalloc’ has not been declared
remote:             , node::smalloc::FreeCallback callback
remote:                     ^
remote:        ../node_modules/nan/nan.h:757:35: error: expected ‘,’ or ‘...’ before ‘callback’
remote:             , node::smalloc::FreeCallback callback
remote:                                           ^
remote:        ../node_modules/nan/nan.h: In function ‘v8::Local<v8::Object> NanNewBufferHandle(char*, size_t, int)’:
remote:        ../node_modules/nan/nan.h:761:50: error: ‘callback’ was not declared in this scope
remote:                 v8::Isolate::GetCurrent(), data, length, callback, hint);
remote:                                                          ^
remote:        ../node_modules/nan/nan.h:761:60: error: ‘hint’ was not declared in this scope
remote:                 v8::Isolate::GetCurrent(), data, length, callback, hint);
remote:                                                                    ^
remote:        ../node_modules/nan/nan.h: In function ‘v8::Local<v8::Object> NanNewBufferHandle(const char*, uint32_t)’:
remote:        ../node_modules/nan/nan.h:768:67: error: call of overloaded ‘New(v8::Isolate*, const char*&, uint32_t&)’ is ambiguous
remote:             return node::Buffer::New(v8::Isolate::GetCurrent(), data, size);
remote:                                                                           ^
remote:        ../node_modules/nan/nan.h:768:67: note: candidates are:
remote:        In file included from ../node_modules/nan/nan.h:25:0,
remote:                         from ../src/binding.cpp:1:
remote:        /app/.node-gyp/4.2.1/src/node_buffer.h:31:40: note: v8::MaybeLocal<v8::Object> node::Buffer::New(v8::Isolate*, v8::Local<v8::String>, node::encoding) <near match>
remote:         NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
remote:                                                ^
remote:        /app/.node-gyp/4.2.1/src/node_buffer.h:31:40: note:   no known conversion for argument 3 from ‘uint32_t {aka unsigned int}’ to ‘node::encoding’
remote:        /app/.node-gyp/4.2.1/src/node_buffer.h:43:40: note: v8::MaybeLocal<v8::Object> node::Buffer::New(v8::Isolate*, char*, size_t) <near match>
remote:         NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
remote:                                                ^
remote:        /app/.node-gyp/4.2.1/src/node_buffer.h:43:40: note:   no known conversion for argument 2 from ‘const char*’ to ‘char*’
remote:        In file included from ../src/binding.cpp:1:0:
remote:        ../node_modules/nan/nan.h: In function ‘v8::Local<v8::Object> NanNewBufferHandle(uint32_t)’:
remote:        ../node_modules/nan/nan.h:772:61: error: could not convert ‘node::Buffer::New(v8::Isolate::GetCurrent(), ((size_t)size))’ from ‘v8::MaybeLocal<v8::Object>’ to ‘v8::Local<v8::Object>’
remote:             return node::Buffer::New(v8::Isolate::GetCurrent(), size);
remote:                                                                     ^
remote:        ../node_modules/nan/nan.h: In function ‘v8::Local<v8::Object> NanBufferUse(char*, uint32_t)’:
remote:        ../node_modules/nan/nan.h:779:12: error: ‘Use’ is not a member of ‘node::Buffer’
remote:             return node::Buffer::Use(v8::Isolate::GetCurrent(), data, size);
remote:                    ^
remote:        ../src/binding.cpp: In function ‘void dispatched_async_uv_callback(uv_async_t*)’:
remote:        ../src/binding.cpp:70:35: warning: ‘void node::FatalException(const v8::TryCatch&)’ is deprecated (declared at /app/.node-gyp/4.2.1/src/node.h:282): Use FatalException(isolate, ...) [-Wdeprecated-declarations]
remote:             node::FatalException(try_catch);
remote:                                           ^
remote:        ../src/binding.cpp: In function ‘void make_callback(uv_work_t*)’:
remote:        ../src/binding.cpp:245:35: warning: ‘void node::FatalException(const v8::TryCatch&)’ is deprecated (declared at /app/.node-gyp/4.2.1/src/node.h:282): Use FatalException(isolate, ...) [-Wdeprecated-declarations]
remote:             node::FatalException(try_catch);
remote:                                           ^
remote:        ../src/binding.cpp: In function ‘void ImportedCallback(const v8::FunctionCallbackInfo<v8::Value>&)’:
remote:        ../src/binding.cpp:365:35: warning: ‘void node::FatalException(const v8::TryCatch&)’ is deprecated (declared at /app/.node-gyp/4.2.1/src/node.h:282): Use FatalException(isolate, ...) [-Wdeprecated-declarations]
remote:             node::FatalException(try_catch);
remote:                                           ^
remote:        make: *** [Release/obj.target/binding/src/binding.o] Error 1
remote:        make: Leaving directory `/app/tmp/cache/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/node-sass/build'
remote:        gyp ERR! build error
remote:        gyp ERR! stack Error: `make` failed with exit code: 2
remote:        gyp ERR! stack     at ChildProcess.onExit (/app/tmp/cache/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/node-sass/node_modules/pangyp/lib/build.js:272:23)
remote:        gyp ERR! stack     at emitTwo (events.js:87:13)
remote:        gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
remote:        gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
remote:        gyp ERR! System Linux 3.13.0-57-generic
remote:        gyp ERR! command "/tmp/build_8d02b095bd87a9217ee50832ded9951d/vendor/node/bin/node" "/app/tmp/cache/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/node-sass/node_modules/pangyp/bin/node-gyp" "rebuild"
remote:        gyp ERR! cwd /app/tmp/cache/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/node-sass
remote:        gyp ERR! node -v v4.2.1
remote:        gyp ERR! pangyp -v v2.3.2
remote:        gyp ERR! not ok
remote:        Build failed
remote:        ember-cli-rails-addon@0.0.12 node_modules/ember-cli-rails-addon
remote:
remote:        ember-cli-inject-live-reload@1.3.1 node_modules/ember-cli-inject-live-reload
remote:
remote:        ember-new-computed@1.0.3 node_modules/ember-new-computed
remote:
remote:        ember-disable-proxy-controllers@1.0.1 node_modules/ember-disable-proxy-controllers
remote:
remote:        ember-export-application-global@1.0.5 node_modules/ember-export-application-global
remote:
remote:        ember-composability@0.1.3 node_modules/ember-composability
remote:
remote:        ember-key-responder@0.4.2 node_modules/ember-key-responder
remote:
remote:        ember-radio-button@1.0.7 node_modules/ember-radio-button
remote:
remote:        active-model-adapter@2.0.1 node_modules/active-model-adapter
remote:
remote:        ember-cli-ic-ajax@0.2.1 node_modules/ember-cli-ic-ajax
remote:        └── ic-ajax@2.0.2
remote:
remote:        ember-cli-htmlbars-inline-precompile@0.2.0 node_modules/ember-cli-htmlbars-inline-precompile
remote:        └── babel-plugin-htmlbars-inline-precompile@0.0.5
remote:
remote:        ember-data@1.13.8 node_modules/ember-data
remote:        └── rsvp@3.1.0
remote:
remote:        ember-cli-dependency-checker@1.1.0 node_modules/ember-cli-dependency-checker
remote:        ├── is-git-url@0.2.0
remote:        ├── semver@4.3.6
remote:        └── chalk@0.5.1 (ansi-styles@1.1.0, escape-string-regexp@1.0.3, supports-color@0.2.0, strip-ansi@0.3.0, has-ansi@0.1.0)
remote:
remote:        ember-legacy-views@0.2.0 node_modules/ember-legacy-views
remote:
remote:        ember-cli-content-security-policy@0.4.0 node_modules/ember-cli-content-security-policy
remote:        └── body-parser@1.14.1 (bytes@2.1.0, content-type@1.0.1, depd@1.1.0, qs@5.1.0, debug@2.2.0, on-finished@2.3.0, http-errors@1.3.1, raw-body@2.1.4, iconv-lite@0.4.12, type-is@1.6.9)
remote:
remote:        ember-cli-htmlbars@0.7.9 node_modules/ember-cli-htmlbars
remote:        ├── ember-cli-version-checker@1.1.4 (semver@4.3.6)
remote:        └── broccoli-filter@0.1.14 (broccoli-writer@0.1.1, promise-map-series@0.2.1, walk-sync@0.1.3, mkdirp@0.3.5, symlink-or-copy@1.0.1, rsvp@3.1.0, quick-temp@0.1.3, broccoli-kitchen-sink-helpers@0.2.9)
remote:
remote:        ember-cli-app-version@0.5.0 node_modules/ember-cli-app-version
remote:        └── git-repo-version@0.3.0 (git-repo-info@1.1.2)
remote:
remote:        ember-modal-dialog@0.7.7 node_modules/ember-modal-dialog
remote:        ├── ember-cli-version-checker@1.1.4 (semver@4.3.6)
remote:        └── ember-wormhole@0.3.4
remote:
remote:        ember-cli-release@0.2.3 node_modules/ember-cli-release
remote:        ├── merge@1.2.0
remote:        ├── semver@4.3.6
remote:        ├── git-tools@0.1.4 (spawnback@1.0.0)
remote:        ├── rsvp@3.1.0
remote:        ├── chalk@1.1.1 (escape-string-regexp@1.0.3, ansi-styles@2.1.0, supports-color@2.0.0, has-ansi@2.0.0, strip-ansi@3.0.0)
remote:        └── moment-timezone@0.3.1 (moment@2.10.6)
remote:
remote:        ember-cli-sri@1.1.0 node_modules/ember-cli-sri
remote:        └── broccoli-sri-hash@1.1.1 (sri-toolbox@0.2.0, rsvp@3.1.0, broccoli-filter@1.2.2, broccoli@0.16.8)
remote:
remote:        broccoli-asset-rev@2.3.0 node_modules/broccoli-asset-rev
remote:        ├── json-stable-stringify@1.0.0 (jsonify@0.0.0)
remote:        ├── rsvp@3.0.21
remote:        ├── broccoli-filter@1.2.2 (copy-dereference@1.0.0, symlink-or-copy@1.0.1, promise-map-series@0.2.1, walk-sync@0.1.3, debug@2.2.0, mkdirp@0.5.1, broccoli-kitchen-sink-helpers@0.2.9, broccoli-plugin@1.2.0)
remote:        └── broccoli-asset-rewrite@1.0.9 (broccoli-filter@0.2.0)
remote:
remote:        ember-cli-materialize@0.18.6 node_modules/ember-cli-materialize
remote:        ├── rsvp@3.1.0
remote:        └── ember-cli-htmlbars@1.0.1 (strip-bom@2.0.0, json-stable-stringify@1.0.0, ember-cli-version-checker@1.1.4, broccoli-persistent-filter@1.1.0)
remote:
remote:        ember-cli-qunit@1.0.3 node_modules/ember-cli-qunit
remote:        └── broccoli-jshint@1.1.0 (chalk@0.4.0, json-stable-stringify@1.0.0, mkdirp@0.4.2, findup-sync@0.1.3, broccoli-persistent-filter@1.1.0, jshint@2.8.0)
remote:
remote:        ember-cli-uglify@1.2.0 node_modules/ember-cli-uglify
remote:        └── broccoli-uglify-sourcemap@1.0.1 (source-map-url@0.3.0, walk-sync@0.1.3, symlink-or-copy@1.0.1, mkdirp@0.5.1, broccoli-writer@0.1.1, uglify-js@2.5.0, lodash-node@2.4.1)
remote:
remote:        ember-cli-babel@5.1.5 node_modules/ember-cli-babel
remote:        ├── clone@1.0.2
remote:        ├── ember-cli-version-checker@1.1.4 (semver@4.3.6)
remote:        ├── resolve@1.1.6
remote:        ├── broccoli-funnel@0.2.10 (array-equal@1.0.0, symlink-or-copy@1.0.1, walk-sync@0.2.5, debug@2.2.0, minimatch@2.0.10, mkdirp@0.5.1, broccoli-plugin@1.2.0)
remote:        └── broccoli-babel-transpiler@5.4.5 (clone@0.2.0, json-stable-stringify@1.0.0, broccoli-persistent-filter@1.1.0, broccoli-merge-trees@0.2.3, babel-core@5.8.25)
remote:
remote:        ember-cli-sass@3.3.1 node_modules/ember-cli-sass
remote:        ├── merge@1.2.0
remote:        ├── ember-cli-version-checker@1.1.4 (semver@4.3.6)
remote:        ├── broccoli-merge-trees@0.2.3 (symlink-or-copy@1.0.1, debug@2.2.0, broccoli-plugin@1.2.0)
remote:        └── broccoli-sass-source-maps@0.7.0 (mkdirp@0.3.5, broccoli-caching-writer@0.5.5, node-sass@2.1.1)
remote:
remote:        ember-cli@1.13.8 node_modules/ember-cli
remote:        ├── process-relative-require@1.0.0
remote:        ├── clean-base-url@1.0.0
remote:        ├── silent-error@1.0.0
remote:        ├── escape-string-regexp@1.0.3
remote:        ├── broccoli-unwatched-tree@0.1.1
remote:        ├── pleasant-progress@1.1.0
remote:        ├── is-git-url@0.2.3
remote:        ├── broccoli-writer@0.1.1
remote:        ├── isbinaryfile@2.0.4
remote:        ├── ember-cli-copy-dereference@1.0.0
remote:        ├── abbrev@1.0.7
remote:        ├── promise-map-series@0.2.1
remote:        ├── bower-endpoint-parser@0.2.2
remote:        ├── js-string-escape@1.0.0
remote:        ├── node-modules-path@1.0.1
remote:        ├── ember-cli-preprocess-registry@1.0.3
remote:        ├── walk-sync@0.1.3
remote:        ├── strip-ansi@2.0.1 (ansi-regex@1.1.1)
remote:        ├── exit@0.1.2
remote:        ├── broccoli-sane-watcher@1.1.4 (broccoli-slow-trees@1.1.0)
remote:        ├── semver@4.3.6
remote:        ├── symlink-or-copy@1.0.1 (copy-dereference@1.0.0)
remote:        ├── debug@2.2.0 (ms@0.7.1)
remote:        ├── temp@0.8.1 (rimraf@2.2.8)
remote:        ├── rsvp@3.1.0
remote:        ├── http-proxy@1.11.2 (eventemitter3@1.1.1, requires-port@0.0.1)
remote:        ├── broccoli-filter@0.1.14 (mkdirp@0.3.5)
remote:        ├── broccoli-kitchen-sink-helpers@0.2.7 (mkdirp@0.3.5)
remote:        ├── merge-defaults@0.2.1 (lodash@2.4.2)
remote:        ├── findup-sync@0.2.1 (glob@4.3.5)
remote:        ├── rimraf@2.3.2 (glob@4.5.3)
remote:        ├── compression@1.6.0 (bytes@2.1.0, on-headers@1.0.1, vary@1.1.0, compressible@2.0.6, accepts@1.3.0)
remote:        ├── portfinder@0.4.0 (async@0.9.0, mkdirp@0.5.1)
remote:        ├── cpr@0.4.3 (graceful-fs@4.1.2, mkdirp@0.5.1, rimraf@2.4.3)
remote:        ├── github@0.2.4 (mime@1.3.4)
remote:        ├── yam@0.0.18 (fs-extra@0.16.5, lodash.merge@3.3.2)
remote:        ├── sane@1.3.0 (watch@0.10.0, minimist@1.2.0, exec-sh@0.2.0, minimatch@0.2.14, fb-watchman@1.6.0, walker@1.0.7)
remote:        ├── broccoli-merge-trees@0.2.2 (broccoli-plugin@1.2.0)
remote:        ├── broccoli-funnel@0.2.7 (walk-sync-matcher@0.2.2, mkdirp@0.5.1, broccoli-plugin@1.2.0)
remote:        ├── ember-router-generator@1.1.0 (recast@0.9.18)
remote:        ├── broccoli-es3-safe-recast@2.0.0 (es3-safe-recast@2.0.1)
remote:        ├── markdown-it@4.3.0 (linkify-it@1.2.0, mdurl@1.0.1, uc.micro@1.0.0, entities@1.1.1, argparse@1.0.2)
remote:        ├── broccoli-es6modules@1.2.2 (mkdirp@0.5.1, esperanto@0.7.4)
remote:        ├── core-object@0.0.2 (lodash-node@2.4.1)
remote:        ├── markdown-it-terminal@0.0.2 (ansi-styles@2.1.0, cli-table@0.3.1, cardinal@0.5.0, lodash-node@3.10.1)
remote:        ├── broccoli-caching-writer@1.1.0 (broccoli-read-compat@0.1.3, core-object@1.1.0, lodash-node@3.10.1)
remote:        └── broccoli-sourcemap-concat@1.1.5 (mkdirp@0.5.1, lodash.uniq@3.2.2, fast-sourcemap-concat@0.2.6, lodash-node@2.4.1, broccoli-caching-writer@2.0.4)
remote: bower active-model-adapter#~2.0.1     invalid-meta active-model-adapter is missing "main" entry in bower.json
remote: bower active-model-adapter#~2.0.1     invalid-meta active-model-adapter is missing "ignore" entry in bower.json
remote: bower ember#2.0.2                     invalid-meta ember is missing "ignore" entry in bower.json
remote: bower ember#>=1.4 <2                  invalid-meta ember is missing "ignore" entry in bower.json
remote:
remote:        Please note that,
remote:            ember-cli-shims#0.0.3, ember-load-initializers#0.1.5 depends on ember#>=1.4 <2 which resolved to ember#1.13.10
remote:            frontend depends on ember#2.0.2 which resolved to ember#2.0.2
remote:            ember-resolver#0.1.21 depends on ember#> 1.5.0-beta.3 which resolved to ember#2.0.2
remote:            ember-data#2.0.1 depends on ember#^2.0.0 which resolved to ember#2.0.2
remote:        Resort to using ember#^2.0.0 which resolved to ember#2.0.2
remote:        Code incompatibilities may occur.
remote:
remote: bower jquery                      extra-resolution Unnecessary resolution: jquery#>=2.1.1
remote: -----> Cleaning up node-gyp and npm artifacts
remote: -----> Building Ember CLI application production distribution
remote:        `libsass` bindings not found. Try reinstalling `node-sass`?
remote:        Error: `libsass` bindings not found. Try reinstalling `node-sass`?
remote:            at getBinding (/app/tmp/cache/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/node-sass/lib/index.js:22:11)
remote:            at Object.<anonymous> (/app/tmp/cache/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/node-sass/lib/index.js:188:23)
remote:            at Module._compile (module.js:435:26)
remote:            at Object.Module._extensions..js (module.js:442:10)
remote:            at Module.load (module.js:356:32)
remote:            at Function.Module._load (module.js:311:12)
remote:            at Module.require (module.js:366:17)
remote:            at require (module.js:385:17)
remote:            at Object.<anonymous> (/app/tmp/cache/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/index.js:5:12)
remote:            at Module._compile (module.js:435:26)
remote:
remote:  !     Push rejected, failed to compile Ember CLI app
remote:
remote: Verifying deploy...
remote:
remote: !	Push rejected to *****.
remote:
To ******
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to '*******'
```

```
git:(master) heroku info
=== octappus
Git Url:       ********
Web Url:       *******
Dynos:         {}
Owner:         *******
Region:        us
Repo Size:     0 B
Slug Size:     0 B
Stack:         cedar-14
```
